lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -2190,6 +2190,11 @@ taggedSumTypeOptions base opts = base
     { sumEncoding = TaggedObject (_tagFieldName opts) (_contentsFieldName opts)
     }
 
+explicitNothingRecordTypeOptions :: Aeson.Options
+explicitNothingRecordTypeOptions = defaultRecordTypeOptions
+    { omitNothingFields = False
```

Comment:
Not sure why this is enabled by default…

"Is the `mary` key not present because the wallet is not supporting `mary`, or because `mary` is simply not enabled yet?"

Explicit `null` for the latter case seems much clearer.

Reply:
I agree that the absence of a field doesn't have the same meaning as a `null` value. I think that in the context of our API indeed, we most likely always want fields corresponding to Maybe values to be present :thinking: